### DISCUSSION TITLE
Dev db

### DISCUSSION THREAD

**Discussion by TimGeyssens** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#13] [General]
So what would be the best way to set this up? Just everyone using a local db? And then use uSync to handle the items that have changes between commits?

**Comment by TimGeyssens** [comment]
Currently just a blank v8 sql ce db... folks can then use usync for the other stuff... (run import on first setup)

**Comment by CarlSargunar** [comment]
Also do we want to commit the models builder files in addition to the dummy database? 

They're not currently ignored from what I can see, so it may be worth commiting - depending on how we end up using them in the site

**Reply by TimGeyssens** [reply]
Sure generated model make sense...otherwise project won't build...
